### PR TITLE
Feature/track challenge timeout cx 2516

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
In order to work out the challenge request latency, we need to track the time elapsed from challenge request until the challenge is loaded.

# Solution
Send an event to woopra when the request is sent and when a response is received (success or failure). As a property, send the time elapsed from the request till the response is received.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://476-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Liveness/index.js [hunk 1]
```diff
@@ -39,22 +41,27 @@ export default class Liveness extends Component<CameraType, State> {
...
     this.setState({ challenges: challenge, id, hasLoaded: true })
+    sendScreen(['face_video_challenge_loaded'], {challenge_loading_time: this.challenge_loading_time()})
   }
 
   handleError = () => {
     this.setState({ hasLoaded: true, hasError: true })
+    sendScreen(['face_video_challenge_load_failed'], {challenge_loading_time: this.challenge_loading_time()})
   }
 
+  challenge_loading_time = () => currentMilliseconds() - this.state.challengeRequestedAt
```
Comment (elhesuu) [inline src/components/Liveness/index.js]: Could we use camelCase instead of snake_case here?

Comment (stefaniacardenas) [reply src/components/Liveness/index.js]: Yes :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-476-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://476-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**